framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss

```diff
@@ -11,8 +15,7 @@
   flex-grow: 1;
   list-style: none;
   margin: 0;
-  padding-left: 0;
-  transition: 0.2s;
```

Comment:
Good catch, the missing background was a twig mistake. 

I removed the transition intentionally because it was useless (or ugly 😄)